src/client/src/apps/MainRoute/widgets/spotTile/spotTileDataReducer.ts

```diff
@@ -3,9 +3,8 @@ import { SpotTileActions, TILE_ACTION_TYPES } from './actions'
 import { PriceMovementTypes } from './model/priceMovementTypes'
 import { SpotTileData } from './model/spotTileData'
 
-export interface SpotTileState {
-  [currencyPair: string]: SpotTileData
-}
+// we want to let compiler know that values for certain keys might be missing
+export type  SpotTileState = { [currencyPair: string]: SpotTileData | undefined }
```

Comment:
this is where `| undefined` was added to point out that there might be missing `SpotTileData ` for certain currency

Reply:
minor: This could be defined using the Record<> type

```
export type  SpotTileState = Record<string, SpotTileData | undefined>
```

Reply:
done